### ISSUE TITLE
Missing locale texts

### ISSUE THREAD

**Issue by jykesaviaGNH** [apluslms/a-plus#1378]
Tested on version 1.24.1
Some _("labels") are not defined in django.po files:

```
"LABEL_SUBMISSION_TAGS_HELPTEXT" NOT in FI
"LABEL_SUBMISSION_TAGS_HELPTEXT" NOT in EN
"MODEL_NAME_SUBMISSION_TAG" NOT in FI
"MODEL_NAME_SUBMISSION_TAG" NOT in EN
"MODEL_NAME_SUBMISSION_TAG_PLURAL" NOT in FI
"MODEL_NAME_SUBMISSION_TAG_PLURAL" NOT in EN
"LABEL_SUBMISSION_TAG" NOT in FI
"LABEL_SUBMISSION_TAG" NOT in EN
"LABEL_SUBMISSION_TAGS" NOT in FI
"LABEL_SUBMISSION_TAGS" NOT in EN

```

**Comment by ihalaij1** [comment]
Thanks!

What command did you use to list these?
We could automate the checking of these so this will not happen in the future.

**Comment by ihalaij1** [comment]
Closing. This was fixed by PR #1386.
